src/rh-demo.ts

```diff
@@ -0,0 +1,68 @@
+import { css, LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+
+export class RhDemo extends LitElement {
```

Comment:
This is a component that we might be able to reuse.  It just displays the source code below a preview.

Reply:
absolutely. :)